cstore_reader.c

```diff
@@ -30,7 +30,11 @@
 #include "storage/fd.h"
 #include "utils/memutils.h"
 #include "utils/lsyscache.h"
+#if PG_VERSION_NUM >= 90500
```

Comment:
I would rather prefer having a separate header file doing this check instead of polluting cstore_reader with postgresql release numbers